### PR TITLE
feat: #82 add image component

### PR DESCRIPTION
**Pull request by Pivodur**
### Overview
added an image component

### Changes description
- image component was implemented 
- storybook for image was implmented

### CODE REVIEW

File: mw-webapp/src/component/image/Image.stories.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  tile: "Image",
+  component: Image,
+  parameters: {layout: "centered"},
+  tags: ["autodocs"],
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {args: {src: "https://picsum.photos/200/300", alt: "Image"}};
```
Comment (tritonJS826) [inline mw-webapp/src/component/image/Image.stories.ts:14]: wow) can you guarantee that this image will be exist tomorrow? What happened with preview in story book if image disappeared? it's gonna surprise the developers a little bit.

Comment (tritonJS826) [reply mw-webapp/src/component/image/Image.stories.ts:14]: why alt - image? does it semantically correct?

File: mw-webapp/src/component/image/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   * Image alt text
+   */
+  alt: string;
+  /**
+   * Additional custom class name for the component
+   */
+  className?: string;
+}
+
+export const Image: React.FC<ImageProps> = (props: ImageProps) => {
```
Comment (tritonJS826) [inline mw-webapp/src/component/image/Image.tsx:19]: Please read docs again, all public members should be documented (all exported functions, variables etc.
)

**Review by tritonJS826 — CHANGES_REQUESTED** [review]
Good luck)

File: mw-webapp/src/component/image/Image.stories.ts (hunk 1 above)
Comment (Pivodur) [reply mw-webapp/src/component/image/Image.stories.ts:14]: 1. I can't guarantee that the source of the image will still exist. But what can I use as a preview instead?
2. Do you want me to write more descriptive text like "Default storybook picture"?

Comment (Ekaterina1994) [reply mw-webapp/src/component/image/Image.stories.ts:14]: 1. You can download any picture and use it for story
2. Yes need to add description but it depends on which picture.

File: mw-webapp/src/component/image/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "src/component/image/Image.module.scss";
+
+interface ImageProps {
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.tsx:4]: Add js-doc to interface too

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/image/Image.stories.ts (hunk 1 above)
Comment (Pivodur) [reply mw-webapp/src/component/image/Image.stories.ts:14]: Should I place an image in the same folder of the component, or should I create an 'assets' folder?

Comment (Ekaterina1994) [reply mw-webapp/src/component/image/Image.stories.ts:14]: Create folder for images

**Pull request closed without merging**